### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KonoromiHimaries**

### CODE REVIEW

**Comment by yous** [comment]
Thank you for the PR, @KonoromiHimaries!

But I got an "unable to resolve host address 'subscribe.adblockplus.org'" error. Seeing <https://adblockplus.org/subscriptions>, the link starts with `abp:subscribe`, and it seems to work with Adblock Plus, uBlock Origin, etc. So the "subscribe" link should be on the next to the following titles:
- Adblock Plus
- uBlock Origin
- uBlock

Also can you update the link to as the following?

``` markdown
([Subscribe](abp:subscribe?location=https%3A//github.com/yous/YousList/raw/master/youslist.txt&title=YousList))
```

**Comment by KonoromiHimaries** [comment]
@yous sorry, your link does not work in github.

you can open issue here
https://github.com/github

**Comment by yous** [comment]
Oh, that's right. Sorry for the confusion. Then can you move the links next to "Adblock Plus", "uBlock Origin", and "uBlock"? Also I want `([Subscribe](...))` instead of `[(Subscribe)](...)`. Thanks!

File: README.md [hunk 1]
```diff
@@ -12,24 +12,20 @@ Block filter for [Adblock Plus][], [uBlock Origin][], [uBlock][], [1Blocker][],
...
+### Adblock Plus [(Subscribe)](https://subscribe.adblockplus.org/?location=https://github.com/yous/YousList/raw/master/youslist.txt&title=YousList)
 
 Install the browser plugin of Adblock Plus and add a subscription as 'YousList' with the above URL.
 
-### uBlock Origin
+### uBlock Origin [(Subscribe)](https://subscribe.adblockplus.org/?location=https://github.com/yous/YousList/raw/master/youslist.txt&title=YousList)
 
 Install the browser plugin of uBlock Origin and enable 'KOR: YousList' by checking it.
 
-### uBlock
```
Comment (KonoromiHimaries) [inline README.md]: uBlockAdmin/uBlock@76b89c0

**Comment by smed79** [comment]
@yous it is safe to merge this PR, see https://issues.adblockplus.org/ticket/2208

**Comment by yous** [comment]
@smed79 Thanks for the reference. I can confirm that this works with uBlock Origin, too.

**Pull request merged**